BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -41,34 +30,35 @@ public actor DefaultTabDataStore: TabDataStore {
 
     private func windowURLPath(for windowID: UUID, isBackup: Bool) -> URL? {
         guard let baseURL = fileManager.windowDataDirectory(isBackup: isBackup) else { return nil }
-        let baseFilePath = isBackup ? PathInfo.backup + "_\(windowID.uuidString)" : PathInfo.store + "_\(windowID.uuidString)"
```

Comment:
I am trying to understand, why is this not necessary anymore? 

And also, just realized on the `TabFileManager` `func windowDataDirectory(isBackup: Bool) -> URL?` I would have expected an explanation of the `isBackup` parameter. I am unsure what this does.

Reply:
This is moved into the TabFileManger and altered slightly so the files look nicer in the directory and are easier to look up for debugging. 
The backup flag is to determine which folder they should go into. The backup folder is a slightly older copy of the window data we can use if something goes wrong during a save, such as a crash or battery dying. I'll add a comment.